QPM/SymLinker.Linker/Abstracts/ISymLinkCreator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SymLinker.Linker
```

Comment:
Why use the `.Linker` namespace here? I think you should simply just remove it since it's unnecessary depth for no clearly obvious reason (everything is already under `SymLinker`, why put everything under `SymLinker.Linker`?)